auth-web/src/components/auth/staff/continuation-application/ContinuationApplicationTable.vue

```diff
@@ -97,60 +158,219 @@
 </template>
 
 <script lang="ts">
-import { defineComponent, reactive } from '@vue/composition-api'
-import BusinessServices from '@/services/business.services'
-import { ContinuationReviewIF } from '@/models/continuation-review'
+import { ContinuationReviewIF, ReviewFilterParams } from '@/models/continuation-review'
+import {
+  DEFAULT_DATA_OPTIONS,
+  cachePageInfo,
+  getAndPruneCachedPageInfo,
+  getPaginationOptions,
+  hasCachedPageInfo
+} from '@/components/datatable/resources'
+import { PropType, computed, defineComponent, reactive, ref, toRefs, watch } from '@vue/composition-api'
+import CommonUtils from '@/util/common-util'
+import ConfigHelper from '@/util/config-helper'
+import { DataOptions } from 'vuetify'
+import { SessionStorageKeys } from '@/util/constants'
+import debounce from '@/util/debounce'
+import moment from 'moment'
+import { useI18n } from 'vue-i18n-composable'
+import { useStaffStore } from '@/stores/staff'
 
 export default defineComponent({
   name: 'ContinuationApplicationTable',
-  setup () {
+  props: {
+    reviewSessionStorageKey: {
+      type: String as PropType<SessionStorageKeys>,
+      default: SessionStorageKeys.ReviewSearchFilter
+    },
+    reviewPaginationNumberOfItemsKey: {
+      type: String as PropType<SessionStorageKeys>,
+      default: SessionStorageKeys.ReviewPaginationNumberOfItems
+    },
+    reviewPaginationOptionsKey: {
+      type: String as PropType<SessionStorageKeys>,
+      default: SessionStorageKeys.ReviewPaginationOptions
+    }
+  },
+  setup (props) {
+    const { t } = useI18n()
+    const staffStore = useStaffStore()
+
     const state = reactive({
       reviews: [] as Array<ContinuationReviewIF>,
       headers: [
-        { text: 'Date Submitted', value: 'date' },
+        { text: 'Date Submitted', value: 'submissionDate' },
         { text: 'NR Number', value: 'nrNumber' },
-        { text: 'Identifying Number', value: 'businessIdentifier' },
+        { text: 'Identifying Number', value: 'identifier' },
         { text: 'Completing Party', value: 'completingParty' },
         { text: 'Status', value: 'status' },
         { text: 'Actions', value: 'action' }
       ],
+      statusTypes: ref<{ text: string; value: string }[]>([
```

Comment:
Why do you need `ref<{...` here? Isn't this a readonly object just like headers, above?

Reply:
Will update it

Reply:
Updated:
![image](https://github.com/user-attachments/assets/20732279-102f-4416-a408-ae787d458da5)